### PR TITLE
Add documentation & correct variables

### PR DESCRIPTION
**Pull request by kassianh**
Followed schema and terraform registry to create documentation.

### CODE REVIEW

**Review by Hanymin — DISMISSED** [review]
Lgtm

**Review by soerenmartius — DISMISSED** [review]
lgtm! good job!

**Comment by thiesen** [comment]
hey @kassianh , great job! maybe we can take the opportunity to use Terradoc v0.0.4 to generate the readme with better attribute anchors?

File: README.md [hunk 1]
```diff
@@ -34,155 +37,253 @@ secure, and production-grade cloud infrastructure.
...
+    DAGS_FOLDER
+    GCP_PROJECT
+    GCS_BUCKET
+    GKE_CLUSTER_NAME
+    SQL_DATABASE
+    SQL_INSTANCE
+    SQL_PASSWORD
+    SQL_PROJECT
+    SQL_REGION
+    SQL_USER
```
Comment (mariux) [inline README.md:114]: please separate by comma and use markdown formatting to make it more readable.

File: README.md [hunk 2]
```diff
@@ -34,155 +37,253 @@ secure, and production-grade cloud infrastructure.
...
+- [**`region`**](#var-region): *(Optional `string`)*<a name="var-region"></a>
 
-## Module Argument Reference
+  The location or Compute Engine region for the environment.
 
-See [variables.tf] and [examples/] for details and use-cases.
+- [**`labels`**](#var-labels): *(Optional `map(string)`)*<a name="var-labels"></a>
 
-### Top-level Arguments
+  User-defined labels for this environment. The labels map can contain no more than 64 entries. Entries of the labels map are UTF8 strings that comply with the following restrictions: Label keys must be between 1 and 63 characters long and must conform to the following regular expression: [a-z]([-a-z0-9]*[a-z0-9])?. Label values must be between 0 and 63 characters long and must conform to the regular expression ([a-z]([-a-z0-9]*[a-z0-9])?)?. No more than 64 labels can be associated with a given environment. Both keys and values must be <= 128 bytes in size.
```
Comment (mariux) [inline README.md:76]: please use markdown formatting and paragraphs to make it more readable (please fix for all docs) you can use github to render the preview of changes to check readability.

**Review by mariux — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kassianh) [reply README.md:114]: Formatted

File: README.md (hunk 2 above)
Comment (kassianh) [reply README.md:76]: Added some formatting to make them more readable.

File: variables.tf [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
+  description = "(Required) Name of the composer environment"
 }
 
-variable "region" {
-  type        = any
-  description = "(Required) The region to deploy resources to"
+variable "project" {
+  type        = string
+  description = "(Optional) The ID of the project in which the resource belongs. If it is not provided, the provider project is used."
 }
```
Comment (mariux) [inline variables.tf:9]: this is required.

File: variables.tf [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
 
-variable "name" {
-  type        = any
-  description = "(Required) Name of the composer environemnt"
+variable "region" {
+  type        = string
+  description = "(Optional) The location or Compute Engine region for the environment."
 }
```
Comment (mariux) [inline variables.tf:14]: this is required.

File: README.tfdoc.hcl [hunk 5]
```diff
@@ -0,0 +1,572 @@
...
+              type        = bool
+              description = <<-END
+                Whether or not to enable Alias IPs in the GKE cluster. If true, a VPC-native cluster is created. Defaults to true if the ip_allocation_policy block is present in config.
+              END
+            }
+
+            attribute "cluster_secondary_range_name" {
+              type        = string
+              description = <<-END
+                The name of the cluster's secondary range used to allocate IP addresses to pods. Specify either cluster_secondary_range_name or cluster_ipv4_cidr_block but not both. For Cloud Composer 1 environments, this field is applicable only when use_ip_aliases is true.
```
Comment (mariux) [inline README.tfdoc.hcl:353]: for readability please quote all variables and things like `true` and `false` and `null`

```suggestion
                The name of the cluster's secondary range used to allocate IP addresses to pods. Specify either `cluster_secondary_range_name` or `cluster_ipv4_cidr_block` but not both. For Cloud Composer 1 environments, this field is applicable only when `use_ip_aliases` is `true`.
```

also true for other occurrences ;)

**Review by mariux — DISMISSED** [review]

**Review by mariux — APPROVED** [review]
lgtm

**Pull request merged**